components/servo/lib.rs

```diff
@@ -412,7 +412,11 @@ where
         } else {
             let dispatcher =
                 Box::new(MainThreadDispatcher::new(compositor_proxy.clone())) as Box<_>;
-            GLContextFactory::current_native_handle(dispatcher, window.gl().get_type())
+            let gl_type = match window.gl().get_type() {
+                gl::GlType::Gl => sparkle::gl::GlType::Gl,
+                gl::GlType::Gles => sparkle::gl::GlType::Gles,
+            };
```

Comment:
Can we re-export GlType to avoid this type conversion?

Reply:
Re-export where? Have gleam depend on sparkle? Have sparkle depend on gleam? Neither solution sounds like an improvement to me.

Reply:
Ah, we're getting the GlType from glutin, oh that makes sense.